pkg/platform/model/buildplanner.go

```diff
@@ -349,6 +350,54 @@ func (bp *BuildPlanner) GetBuildExpression(owner, project, commitID string) (*bu
 	return expression, nil
 }
 
+func (bp *BuildPlanner) CreateProject(owner, project, platformID, language, version string, private bool) (strfmt.UUID, error) {
+	logging.Debug("CreateProject, owner: %s, project: %s, language: %s, version: %s", owner, project, language, version)
+
+	// At this time, there is no way to ask the Platform for an empty buildexpression, so build one
+	// manually and then add a timestamp, platform, and language requirement to it.
+	expr, err := buildexpression.New([]byte(fmt.Sprintf(`
+		{
+			"let": {
+				"runtime": {
+					"solve_legacy": {
+						"at_time": "%s",
+						"build_flags": [],
+						"camel_flags": [],
+						"platforms": ["%s"],
+						"requirements": [],
+						"solver_version": null
+					}
+				},
+				"in": "$runtime"
+			}
+		}
+	`, time.Now().Format(time.RFC3339), platformID)))
```

Comment:
Could this be a convenience function in the `buildexpression` package? This seems like it should live there.